frontend/package.json

```diff
@@ -1,33 +1,42 @@
 {
-  "name": "vue-frontend",
+  "name": "react-frontend",
   "version": "0.0.1",
+  "type": "module",
   "scripts": {
     "dev": "vite --host",
     "dockerbuild": "vite build --mode dockerbuild",
```

Comment:
doesn't this also need tsc? at least dockerfile currently fails with exit code 1